frontend/storybook/src/components/entry/popups/studentcode-paragraph.ts

```diff
@@ -0,0 +1,90 @@
+import "@elements/entry/home/TOSORT/column-details";
+import "@elements/entry/home/TOSORT/column-list";
+import "@elements/core/popups/popups-template";
+import {Color,Size} from "@elements/core/popups/popups-template";
+import {argsToAttrs, deleteNone} from "@utils/attributes";
+import "@elements/core/titles/variants/title-section";
+import "@elements/entry/popup/sections/studentcode-section";
+import {Image} from "@elements/entry/popup/sections/studentcode-section";
+import "@elements/core/dividers/square-divider";
+
+
+const STR_SMALL="small";
+const STR_DARKBLUE="darkblue";
+const STR_TITLE="Enter your student code";
+const STR_PATH="Image_JIG_StudentCode@2x.png";
+const STR_BLUE="blue";
+ 
+
+
+export default {
+    title: 'Popups',
+  }
+
+  interface Args{
+    color: Color,
+    size: Size,
+    image:Image
+  }
+  
+  const DEFAULT_ARGS:Args = {
+    color:"green",
+    size: "large",
+    image:"Illustration_JIG_Sad_1.png.png"
+  }
+  
+  
+ 
+
+
+export const studentCode= (props?:Partial<Args>) => {
+const {...popupProps} = props;
+props = props ? {...DEFAULT_ARGS, ...props} : DEFAULT_ARGS;
+    return `
+ <template-popups  ${argsToAttrs(deleteNone(popupProps))}>
+ <studentcode-section>
+ <title-section titlecolor="${STR_DARKBLUE}" title="${STR_TITLE}" size="${STR_SMALL}" slot="title"></title-section>
+ <square-divider colorborder="${STR_SMALL}" size="${STR_BLUE}" slot="square"></square-divider>
```

Comment:
size="${STR_BLUE}" seems very weird... one is a size and the other is a color. it's possible some other mistakes like this crept in. Please double-check in other places where this might have happened too